Eevee.Sleep.Bot/Controllers/Mongo/DiscordRoleController.cs

```diff
@@ -0,0 +1,36 @@
+using Eevee.Sleep.Bot.Models;
+using MongoDB.Driver;
+
+namespace Eevee.Sleep.Bot.Controllers.Mongo;
+
+public static class DiscordRoleContoller {
+    public static async Task SaveRoles(ulong userId, ulong[] roles) {
+        await MongoConst.RoleCollection.UpdateOneAsync(
```

Comment:
Return the `Task` of `UpdateOneAsync()` and remove the `async` from the method header.

https://stackoverflow.com/a/38017167/11571888

Reply:
Fixed!
[b7888c9](https://github.com/RaenonX-PokemonSleep/eevee-sleep-discord-bot/pull/25/commits/b7888c9f5cf66ca9b630830aa447b2751aed6dfb)